__tests__/fixtures/fakeCourse.js

```diff
@@ -8,9 +8,9 @@ module.exports = function makeFakeCourse() {
 		category: 'sewing',
 		published: false,
 		sections: [],
-		creator: [],
 		difficulty: 1,
-		time: 1,
+		estimatedHours: 1,
 		rating: 5,
+    numOfSubscriptions: 0,
```

Comment:
Very minor comment, "numOfSubsricptions" does not seem to be formatted correcly

Reply:
No problem, I fixed it anyway 😎👍